apps/team/src/pages/location/[location].tsx

```diff
@@ -1,8 +1,10 @@
 import { useRouter } from "next/router";
-import TeamNav from "../index";
+import TeamNav, { ServerProps } from "../index";
 import { Spinner } from "@artsy/palette";
-import { normalizeParam } from "lib/url";
-import { NoResults } from "components/team/NoResults";
+import { normalizeParam } from "@artsy-studio/utils";
```

Comment:
Would it make sense in moving things like this to the app level to keep surface layer minimal?

Reply:
Moved to the app level